docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
 jax>=0.1.65
 jaxlib>=0.1.45
-funsor>=0.1.2
+git+git://github.com/pyro-ppl/funsor@master#egg=funsor
```

Comment:
Note that we need funsor's master branch to build the documentation. We can probably remove this on next release of funsor.